source/IAccessibleHandler/internalWinEventHandler.py

```diff
@@ -114,14 +127,20 @@ def winEventCallback(handle, eventID, window, objectID, childID, threadID, times
 			global _deferUntilForegroundWindow, _foregroundDefers
 			_deferUntilForegroundWindow = window
 			_foregroundDefers = 0
+			if isMSAADebugLoggingEnabled():
+				log.debug("Recording foreground winEvent defer")
 		if windowClassName == "MSNHiddenWindowClass":
 			# HACK: Events get fired by this window in Windows Live Messenger 2009 when it starts. If we send a
 			# WM_NULL to this window at this point (which happens in accessibleObjectFromEvent), Messenger will
 			# silently exit (#677). Therefore, completely ignore these events, which is useless to us anyway.
 			return
+		if isMSAADebugLoggingEnabled():
+			log.debug(
+				f"Adding winEvent to limitor: {getWinEventLogInfo(window, objectID, childID, eventID)}"

```

Comment:
limitOr should be limitEr

```suggestion
				f"Adding winEvent to limiter: {getWinEventLogInfo(window, objectID, childID, eventID)}"

```